web/controllers/form_settings_controller.ex

```diff
@@ -5,11 +5,10 @@ defmodule Formerer.FormSettingsController do
 
   def edit(conn, %{"form_id" => id}) do
     case current_user(conn) |> get_user_form(id) do
-      nil ->
-        send_resp(conn, 404, "Not Found")
-      form ->
+      { :error, error } ->
```

Comment:
Can we have the normal execution path first and the error case second. I know I had them wrong way around for matching purposes initially but now that mess is fixed we can have them the "correct" way around again 😉